### ISSUE TITLE
Add CONTRIBUTORS list?

### ISSUE THREAD

**Issue by kyleniemeyer** [pr-omethe-us/ChemKED-database#10]

**Comment by bryanwweber** [comment]
Seems like this would be captured in the `file-authors` field.

**Comment by kyleniemeyer** [comment]
Right—my thinking was more that the CONTRIBUTOR list could be used to populate a list of names when citing the database, but all the entries in `file-authors` is not easily accessible from GitHub.
